### DISCUSSION TITLE
Noah-MP Irrigation testcase?

### DISCUSSION THREAD

**Discussion by yxue4** [NASA-LIS/LISF#928] [Q&A] [answered]
Dear LIS team,

I was trying to explore the feature of irrigation using the Noah-MP 3.6 - CONUS Testcase as a start within the latest release of LISF-7.4.5-557WW.

The README file says "Download the sample input data from: http://lis.gsfc.nasa.gov/Source/testcases
the testcase corresponding to NOAH-MP IRRIGATION". Unfortunately, I did not see a testcase as mentioned on this page, nor under the tab of "Testcases".

Would you please advise where and how to obtain these sample data in order to run such a test case? Thank you for your help!

Best,
Yuan

**Comment by bmcandr** [comment]
Hi @yxue4,

I believe the URL should actually be: https://portal.nccs.nasa.gov/lisdata_pub/TESTING/testcases/data/

Unfortunately, the system hosting our testcases is down for maintenance today so I'm unable to point you directly to the file right now. I will circle back once the system is back up.

**Reply by yxue4** [reply]
Hi,

Thanks for your quick reply! Now I see the data portal is online, would you mind letting me know where I could obtain the sample input and output for the Noah-MP irrigation testcase?

Thanks,
Yuan

**Reply by bmcandr** [reply]
Unfortunately it appears that the the input files for that testcase are not currently available in a publicly accessible directory on our portal. @jvgeiger manages the testcase portal so I will let him respond to this request.

**Comment by jvgeiger** [comment, marked as answer]
Hello,

I am sorry.  The input and output data for the NoahMP 3.6 irrigation test case are not on our data portal because they are very large and I do not have the resources to stage them there.

**Comment by bmcandr** [comment]
@yxue4, we are looking into whether and how we can share these testcase files with you. Thank you for bringing this to our attention and for your patience.

**Reply by yxue4** [reply]
Hi,

Thanks for your efforts! Please take your time, no rush. In the meantime, I will just explore it by myself.

I noticed there is a parameter file named "maxrootdepth32.txt", but I do not see it included in "https://portal.nccs.nasa.gov/lisdata_pub/data/PARAMETERS/irrigation/". Would you mind kindly let me know where to have access to this file? Thank you!

Best,
Yuan

**Reply by bmcandr** [reply]
Hi Yuan,

Sorry for the delay. Here is the requested file:

[maxrootdepth32.txt](https://github.com/NASA-LIS/LISF/files/7571436/maxrootdepth32.txt)
